docs/apps/MinimalApps/Files/Classification.mdx

```diff
@@ -0,0 +1,12 @@
+---
+sidebar_position: 6
+---
+# Classification 
+
+### There are three types of menus on the side :
+
+- **Favorites** (most used *category* of files)
+- **Pinned** (files that have been pinned through the [File Context Menu (Left-click)](https://www.notion.so/File-Context-Menu-Left-click-84e162b3a94d4d66b9b2ea65fef2ea84))
```

Comment:
You've linked to the Notion here — remember this website is public; the Notion isn't.

Reply:
The solution here to either comment/remove this out, ore make this page

Reply:
> The solution here to either comment/remove this out, or make this page

this is fixed